src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,382 @@
+</Callout>
+
+In order to get the sidecar injected on pods deployed before the operator has been installed, the user needs to manually perform a rollout (restart). New Relic has chosen not to do this automatically in order to prevent unexpected service disruptions and resource usage spikes.
+
+Here's the injection workflow:
+
+![Diagram showing the workflow of sidecar injection](./images/fargate-workflow.svg "Diagram showing the workflow of sidecar injection")
+
+#### Automatic injection installation [#auto-injection-install]
+
+<Callout variant="tip">
+These setup steps are for a generic installation. Before completing these, take a look at the [Configuration](#config-auto) section below to see if you want to modify any aspects of the automatic injection.
+</Callout>
+
+In order to install the operator in charge of injecting the infrastructure sidecar, all you have to do is add the following flags to the the [install bundle command](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#install-k8-helm):
+
+```
+helm upgrade --install newrelic newrelic/nri-bundle \
+--set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+--set global.cluster=K8S_CLUSTER_NAME \
+--set infrastructure.enabled=true \
+--set prometheus.enabled=true \
+--set webhook.enabled=true \
+--set ksm.enabled=false \
+--set kubeEvents.enabled=true \
+<var>--set global.fargate=true</var> \
+<var>--set newrelic-infra-operator.enabled=true</var>
+```
+
+<Callout variant="important">
+In the case of a hybrid cluster (EC2 and Fargate), make sure after deploying that the solution is not selected by any Fargate profiles; otherwise, the `DaemonSet` instances will be stuck in a pending state. Fargate-only environments don't have this limitation because no `DaemonSet` instances are created.
+
+</Callout>
+
+#### Configuration [#config-auto]
+
+You can configure different aspects of the automatic injection. By default, the operator will inject the monitoring sidecar to all pods deployed in Fargate nodes which are not part of a `Job` or a `BatchJob`.
+
+Using configurations, you can change this behavior. For example, you can define selectors to narrow or widen the selection of pods that are injected, assign resources to the operator, and tune the sidecar. Also, you can add other attributes, labels, and environment variables. Please refer to the chart [README.md](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-infra-operator/README.md) and [values.yaml](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-infra-operator/values.yaml).
+
+<Callout variant="tip">
+Injecting the sidecar into deployments scheduled into EC2 nodes may lead to incorrect or duplicate data, especially if those nodes are already being monitored with the `DaemonSet`. Take this into account when designing custom selectors for the automatic injection.
+</Callout>
```

Comment:
```suggestion
<Callout variant="important">
Specifying your own custom injection rules will discard the default ruleset that prevents sidecar injection on pods that are not scheduled in Fargate. Please ensure that your custom rules have the same effect; otherwise on hybrid clusters which also have the `DaemonSet` deployed, pods scheduled in EC2 will be monitored twice leading to incorrect or duplicate data.
</Callout>
```